Gemfile

```diff
@@ -25,3 +25,5 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem "html-proofer"
+
+gem "jekyll-redirect-from"
```

Comment:
Locally, using the same workflow as the deployment playbook, this gem (and the paginate one) need to be put under the `jekyll_plugins` group to work.